scripts/ubuntu-jammy-from-community-gui/50-wayland_errrata.sh

```diff
@@ -105,5 +105,7 @@ apt install -y ubuntu-restricted-extras wl-clipboard
 
 
 #
-# TODO: generate mutter's monitors.xml (only if needed)
+# TODO: generate mutter's monitors.xml
```

Comment:
So from https://firefox-ci-tc.services.mozilla.com/tasks/P-s9ONuKSSWIGcgPge-lJQ/runs/0/logs/public/logs/live_backing.log#L44 we can rely on that it seems:
```
$ cat /etc/xdg/monitors.xml 
<monitors version="2">
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>Virtual-1</connector>
          <vendor>unknown</vendor>
          <product>unknown</product>
          <serial>unknown</serial>
        </monitorspec>
        <mode>
          <width>1920</width>
          <height>1080</height>
          <rate>60.000</rate>
        </mode>
      </monitor>
    </logicalmonitor>
  </configuration>
</monitors>
```

Reply:
@aerickson

Reply:
Added.